graph/src/main/kotlin/Main.kt

```diff
@@ -115,7 +115,7 @@ object Main {
         val project = analyzerResult.project
         val graph = SingleGraph(project.name)
 
-        graph.setStrict(true)
+        graph.isStrict = true
```

Comment:
I deliberately didn't do this to keep it visually aligned with the lines below. Maybe add code to suppress the hint then?

Reply:
I think it's ok that they don't look the same, because they are not the same: This is property setter, the others are function calls. Actually the line below should be a property setter, too, but it isn't because of inconsistent naming in the API.
I prefer a bit of visual inconsistency over adding an annotation.

Reply:
> Actually the line below should be a property setter, too, but it isn't because of inconsistent naming in the API.

Exactly that was my reasoning to keep using the function call here because of the inconsistency with the line below otherwise. But ok, let's not over-do it, esp. as we now gain `apply {}` syntax with your change :wink: